src/content/reference/firmware.md

```diff
@@ -1846,8 +1861,14 @@ Initializes the SPI bus by setting SCK, MOSI, and a user-specified slave-select
 // SYNTAX
 SPI.begin(ss);
 ```
+{{#if photon}}```C++
+// Example of using second port on Photon, with alternate SS pin:
+SPI1.begin(D5);
+```{{/if}}
 
-Where, the parameter `ss` is the SPI device slave-select pin to initialize.  The default pin is `SS (A2)` if no pin is specified.
+Where, the parameter `ss` is the SPI device slave-select pin to initialize.  
+If no pin is specified, the default pin is `SS (A2)`{{#if photon}} for
+`SPI`, or `SS3 (D5)` for `SPI1`{{/if}}.
```

Comment:
@dougalcampbell, this is incorrect until 0.4.6 is released right?

It would be better for us to fix what's the current status since D5 is not the SPI1 default as per now.